src/doc-templates/index.md.jinja2

```diff
@@ -50,7 +50,7 @@ See [here](https://github.com/mapping-commons/sssom/tree/master/examples/schema)
 | Column/Field       | Description                                             | Required    |
 |--------------------|---------------------------------------------------------|-------------|
 {%- for slot in c.slots %}
-{%- set slot_info = schemaview.get_slot(slot) %}
+{%- set slot_info = schemaview.induced_slot(slot, c.name) %}
```

Comment:
It looks like the logic in these lines and some lines above can be reduced using the [class_induced_slots()](https://github.com/linkml/linkml-runtime/blob/main/linkml_runtime/utils/schemaview.py#L1430-L1439) method in SchemaView?

Like if you know the name of the class (mapping/mapping set) for which you want to retrieve a list of induced slots, then you can simply make calls to `class_induced_slots()` and pass _mapping_/_mapping set_ as arguments.